src/waveform/renderers/waveformoverviewrenderer.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <QColor>
+
+#include "waveform/overviewtype.h"
+#include "waveform/waveform.h"
+
+class QPainter;
+class WaveformSignalColors;
+
+class WaveformOverviewRenderer {
+  public:
+    static QImage render(ConstWaveformPointer pWaveform,
```

Comment:
NIT: use a namespace directly instead of all-static class

Reply:
Like `static QImage WaveformOverviewRenderer::render()` etc.?
Or literally a `WaveformOverviewRenderer` namespace?

Reply:
literally a `WaveformOverviewRenderer` namespace (maybe change the naming to match our naming rules for namespaces) and then just free the functions like it was C.